code401Challenges/src/main/java/code401Challenges/hashtable/TreeIntersection.java

```diff
@@ -1,21 +1,57 @@
 package code401Challenges.hashtable;
 
 
-import code401Challenges.tree.Tree;
+import code401Challenges.tree.Node;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
 
 public class TreeIntersection {
-    // Traverse through tree1. Add all values to a hashmap with null value. Then, inorder traverse through tree2.
-    // If node.value of tree2 in hashmap, add node.value to an arrayList. Return that arraList,
-    // since that will only contain values that intersect.
+    // starting code from Michelle Ferreirae's code review on Oct 30, 2019 Code401-Java-d6
+    // Algorithm: Add all values from first tree into a hashmap, and then check to see if any of the
+    // values in the second tree are in the hashmap. If so, add those values to a hashset and return
+    // the hashset, since it will contain only those node values that both trees share.
 
-    // Review Michelle's FrontRow video from October 30th during Code Review.
+    public static Set<Integer> treeIntersection(Node root1, Node root2) {
```

Comment:
You probably want to declare the parameters as Integer nodes if that's what you are expecting. So your method signature would be `public static Set<Integer> treeIntersection(Node<Integer> root1, Node<Integer> root2) {`